src/wipplpy/mst/connection.py

```diff
@@ -0,0 +1,73 @@
+"""
+Define the MSTConnection() class specifically for making MST-MDSplus connection
+requests. 
+"""
+
+# Standard
```

Comment:
Where these packages come from doesn't need to be commented. i.e. get rid of `# Standard` and `# Local`.